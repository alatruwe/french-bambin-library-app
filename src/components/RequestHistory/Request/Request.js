export default function Request(props) {
  return (
    <section>
      <h2>Item: {props.title}</h2>
      <h1>Request object: {props.object}</h1>
      <p>Request message: {props.message}</p>
      <p>Sent: {props.date}</p>
    </section>
  );
}
