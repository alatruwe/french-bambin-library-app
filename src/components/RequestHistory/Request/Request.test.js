import React from "react";
import ReactDOM from "react-dom";
import Request from "./Request";
import renderer from "react-test-renderer";

describe(`Request component`, () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Request />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders the UI as expected", () => {
    const tree = renderer.create(<Request />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
