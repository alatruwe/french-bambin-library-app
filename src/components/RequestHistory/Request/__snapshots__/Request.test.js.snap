// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Request component renders the UI as expected 1`] = `
<section>
  <h2>
    Item: 
  </h2>
  <h1>
    Request object: 
  </h1>
  <p>
    Request message: 
  </p>
  <p>
    Sent: 
  </p>
</section>
`;
